src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let amount = 0;
+
+  const adder = (x) => {
+    if (x !== undefined) {
+      amount += x;
+
+      return adder;
+    } else {
```

Comment:
```suggestion
    }
```
No need in this else here
Function will return something either way